qa/rpc-tests/timestampindex.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014-2015 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#
+# Test timestampindex generation and fetching
+#
+
+import time
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
```

Comment:
I am not 100% sure, but I think this will fail the unused import check.
```suggestion
from test_framework.util import (
    assert_equal,
    connect_nodes,
    initialize_chain_clean,
    start_nodes,
    stop_nodes,
    wait_bitcoinds,
)
```